types/react-scramble/index.d.ts

```diff
@@ -0,0 +1,102 @@
+// Type definitions for react-scramble 0.4
+// Project: https://github.com/cettoana/react-scramble#readme
+// Definitions by: fes300 <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-scramble/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me